e2e-tests/playwright/tests/functional/channels/drafts/drafts_on_deleted_message.spec.ts

```diff
@@ -0,0 +1,181 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {PreferenceType} from '@mattermost/types/preferences';
+import {Channel} from '@mattermost/types/channels';
+import {UserProfile} from '@mattermost/types/users';
+import {Post} from '@mattermost/types/posts';
+
+import {test, expect} from '@e2e-support/test_fixture';
+import {createRandomPost} from '@e2e-support/server/post';
+import {createRandomChannel} from '@e2e-support/server';
+
+test('MM-T5435_1 Gloabl Drafts link in sidebar should be hidden when another user deleted root post and user removes the deleted post ', async ({
+    pw,
+    pages,
+}) => {
+    const {adminClient, team, adminUser, user, userClient} = await pw.initSetup();
+
+    if (!adminUser) {
+        throw new Error('Failed to create admin user');
+    }
+
+    try {
+        // # Set preferences to disable tooltips for drafts and threads
+        await userClient.savePreferences(user.id, createNoTooltipForDraftsAndThreadsPreferences(user.id));
+        await adminClient.savePreferences(adminUser.id, createNoTooltipForDraftsAndThreadsPreferences(adminUser.id));
+    } catch (error) {
+        throw new Error('Failed to set preferences');
+    }
+
+    let channel: Channel;
+    let adminPost: Post;
+    try {
+        // # Create a new channel
+        channel = await adminClient.createChannel(
+            createRandomChannel({
+                teamId: team.id,
+                name: 'test_channel',
+                displayName: 'Test Channel',
+                unique: true,
+            })
+        );
+
+        // # Add admin and user to the channel
+        await userClient.addToChannel(user.id, channel.id);
+        await adminClient.addToChannel(adminUser.id, channel.id);
```

Comment:
To not complicate the setup, let's use one of the existing default channels:
```javascript
channel = await adminClient.getChannelByName(team.id, 'town-square');
```

Reply:
Done ✅